projects/core/src/lib/time/time-step-selector/time-step-selector.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="container mat-typography" fxLayout="row" fxLayoutAlign="space-between center">
+<div *ngIf="displayTimeLine()" class="container mat-typography" fxLayout="row" fxLayoutAlign="space-between center">
```

Comment:
If you do it like this, the whole time information is gone.

I guess it makes more sense to only hide the slider component but not the time information.

Reply:
I would just show the year, not "Point in Time" before that. It is the same info as with the slider.